src/lib/reporters/Html.ts

```diff
@@ -3,6 +3,32 @@ import Reporter, { eventHandler, ReporterProperties } from './Reporter';
 import Test from '../Test';
 import Suite from '../Suite';
 
+const location = window.location;
+
+function getFullName(test: Suite | Test): string {
+	let name = test.name;
+	while (test.parent && test.parent.name) {
+		test = test.parent;
+		name = `${test.name} - ${name}`;
+	}
+	return name;
+}
+
+function createLinkNode(test: Suite | Test): Element {
+	const a = document.createElement('a');
+	let search = location.search;
+
+	if (search) {
+		search = search.slice(1).split('&').filter(function (el) {
```

Comment:
For consistency, prefer arrow functions for simple callbacks.

Reply:
done